src/ui/components/Footer/Footer.jsx

```diff
@@ -15,9 +19,9 @@ class Footer extends React.Component {
                     <a href="https://www.idealo.co.uk/terms.html">About/Terms & Conditions</a>
                 </div>
                 <div className={s.SocialMedia}>
-                    <a href="https://www.facebook.com/idealo/"><img src="https://www.vectorico.com/wp-content/uploads/2018/02/Facebook-Icon-Dark.png" alt="Facebook Logo" height={40} width={40}/></a>
-                    <a href="https://www.instagram.com/idealo/?hl=en"><img src="https://www.vectorico.com/wp-content/uploads/2018/02/Instagram-Icon.png" alt="Instagram Logo"height={40} width={40}/></a>
-                    <a href="https://twitter.com/idealo_uk"><img src="https://www.vectorico.com/wp-content/uploads/2018/02/Twitter-Social-Icon.png" alt="Twitter Logo" height={40} width={40}/></a>
+                    <a href="https://www.facebook.com/idealo/"><IconFacebook class={s.icon}/></a>
+                    <a href="https://www.instagram.com/idealo/?hl=en"><IconInstagram class={s.icon}/></a>
+                    <a href="https://twitter.com/idealo_uk"><IconTwitter class={s.icon}/></a>
```

Comment:
i think `class` should rename to `className = {...}`

Reply:
it worked with class to but I changed it now